pom.xml

```diff
@@ -26,7 +26,7 @@
 		<redis.lettuce.version>5.2.2.RELEASE</redis.lettuce.version>
 		<opensrp.updatePolicy>always</opensrp.updatePolicy>
 		<nexus-staging-maven-plugin.version>1.5.1</nexus-staging-maven-plugin.version>
-		<opensrp.core.version>2.4.3-SNAPSHOT</opensrp.core.version>
+		<opensrp.core.version>2.4.4-SNAPSHOT</opensrp.core.version>
```

Comment:
Change to 2.1.26

Reply:
2.1.25 is not tagged yet we can tag this there too